metaswitch/common/alarm_severities.py

```diff
@@ -0,0 +1,51 @@
+# @file alarm_severities.py
+#
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2015 Metaswitch Networks Ltd
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version, along with the "Special Exception" for use of
+# the program along with SSL, set forth below. This program is distributed
+# in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details. You should have received a copy of the GNU General Public
+# License along with this program.  If not, see
+# <http://www.gnu.org/licenses/>.
+#
+# The author can be reached by email at clearwater@metaswitch.com or by
+# post at Metaswitch Networks Ltd, 100 Church St, Enfield EN2 6BQ, UK
+#
+# Special Exception
+# Metaswitch Networks Ltd  grants you permission to copy, modify,
+# propagate, and distribute a work formed by combining OpenSSL with The
+# Software, or a work derivative of such a combination, even if such
+# copying, modification, propagation, or distribution would otherwise
+# violate the terms of the GPL. You must comply with the GPL in all
+# respects for all of the code used other than OpenSSL.
+# "OpenSSL" means OpenSSL toolkit software distributed by the OpenSSL
+# Project and licensed under the OpenSSL Licenses, or a work based on such
+# software and licensed under the OpenSSL Licenses.
+# "OpenSSL Licenses" means the OpenSSL License and Original SSLeay License
+# under which the OpenSSL Project distributes the OpenSSL toolkit software,
+# as those licenses appear in the file LICENSE-OPENSSL.
+
+# This module defines the shared alarm severity definitions used by the
+# different parts of the alarm code. It is not possible to put these in
+# alarms.py as the alarm_parser needs access to these, and cannot import
+# alarms as it needs to run too early in the build process.
+
+# valid severity levels - this should be kept in sync with the
+# list in alarmdefinition.h in cpp-common
+# alarms are stored in itu alarm table using the severities below.
+# alarm model table stores alarms according to their state. the
+# mapping between state and severity is described in rfc 3877
+# section 5.4: https://tools.ietf.org/html/rfc3877#section-5.4.
+CLEARED = 1
+INDETERMINATE = 2
+CRITICAL = 3
+MAJOR = 4
+MINOR = 5
+WARNING = 6
```

Comment:
I've fixed up the comment. I agree that comments blur into the copyright statement. I'll switch to docstrings.